tests/test_validator.py

```diff
@@ -169,6 +186,7 @@ def test_complete_config(self, mock_out, mock_err):
         output = "\n".join(errors)
         self.assertEqual(['postgresql.bin_dir', 'raft.bind_addr', 'raft.self_addr'], parse_output(output))
 
+    @patch('socket.getaddrinfo', getaddrinfo_side_effect)
```

Comment:
```suggestion
    @patch('socket.getaddrinfo', mock_getaddrinfo)
```

And actually it would be better to move this decorator to the `TestValidator` class.